__tests__/integration/utils/test-utils-integration.js

```diff
@@ -252,20 +183,36 @@ export class TestUtils {
    */
   static async startWallet(walletObj, options = {}) {
     let response;
+
+    WalletBenchmarkUtil.informWalletEvent(
+      walletObj.walletId,
+      WALLET_EVENTS.startRequest,
+      { multisig: walletObj.multisig }
+    );
+
     // Request the Wallet start
     if (walletObj.words) {
       response = await request
         .post('/start')
-        .send({ seed: walletObj.words, 'wallet-id': walletObj.walletId });
+        .send({
+          seed: walletObj.words,
+          'wallet-id': walletObj.walletId,
+          preCalculatedAddresses: walletObj.addresses
```

Comment:
I think the description in the typedef of `WalletData. addresses` is not good. We should describe better the use of it.

Reply:
I did an overhaul of this `typedef` on [5e3955c](https://github.com/HathorNetwork/hathor-wallet-headless/pull/189/commits/5e3955c29b0eb8b8789ede4620ecafc5a3980891). Its now clearer and better formatted.